ppconsul.sh

```diff
@@ -19,8 +19,19 @@ make ${JOBS:+-j$JOBS}
 #make install
 mkdir $INSTALLROOT/lib
 mkdir $INSTALLROOT/include
-cp $BUILDROOT/$PKGNAME/output/libjson11.so $INSTALLROOT/lib/
-cp $BUILDROOT/$PKGNAME/output/libppconsul.so $INSTALLROOT/lib/
+
+LIB_JSON=so
```

Comment:
Why do you need two of them? How about using simply `$SONAME` like done elsewhere?